src/parser/Statement.ts

```diff
@@ -1548,6 +1559,8 @@ export class ClassStatement extends Statement implements TypedefProvider, Member
                 this.memberMap[statement?.name?.text.toLowerCase()] = statement;
             }
         }
+        this.symbolTable.identifier = `Class (symbols) ${name?.text}`;
+        this.memberTable.identifier = `Class (members) ${name?.text}`;
```

Comment:
Put the `(symbols)` and `(members)` at the end. 
```suggestion
        this.symbolTable.identifier = `Class ${name?.text} (symbols) `;
        this.memberTable.identifier = `Class ${name?.text} (members)`;
```